tests/data/product_stability/sle15.yml

```diff
@@ -18,12 +20,16 @@ cpes:
 cpes_root: ../../shared/applicability
 dconf_gdm_dir: gdm.d
 faillock_path: /var/run/faillock
+families:
+- suse
 full_name: SUSE Linux Enterprise 15
 gid_min: 1000
 groups: {}
 grub2_boot_path: /boot/grub2
 grub2_uefi_boot_path: /boot/grub2
+grub_helper_executable: grubby
```

Comment:
For SLE15 we support grubby, but by _default_ has grub2-mkconfig, so we are better off with that helper. I guess that change should be valid also regarding 10-grub.yml properties file

Reply:
Well, isn't that great? The test clearly exposes the wrong assumption, so one can fix it easily, and that's it - there doesn't have to be any other place in the project to worry about product-approach mapping.

Reply:
This point mentioned by @teacup-on-rockingchair was already addressed in 1ca395cb9d61615dc31f23fba89101c58dcfc6d6 . So we believe we are good to merge this PR and have it already in the 0.1.69 release.